app/views/home/partials/_hero.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="section__header flex flex-center white gutters">
   <div class="section__header-content">
-    <span class="logo--wcmc margin-center"></span>
-    <h1><%= data['title'] %></h1>
+    <div class="logo-container">
+      <span class="logo logo--wcmc"></span>
+      <span class="logo logo--unep"></span>
+    </div>
+    <h1><%= data['title'].html_safe %></h1>
```

Comment:
I can't see the logos on my local and also the container seems to occupy too much space(see screen), could you please have a look?

Reply:
![Screenshot from 2023-01-25 23-11-14](https://user-images.githubusercontent.com/40596660/214713057-35c0b130-bd28-4dae-8687-31566a0b0d41.png)